packages/dendron-next-server/pages/vscode/calendar-view.tsx

```diff
@@ -0,0 +1,187 @@
+import {
+  createLogger,
+  engineSlice,
+  ideHooks,
+  postVSCodeMessage,
+} from "@dendronhq/common-frontend";
+import {
+  DMessageSource,
+  CalendarViewMessageType,
+  DNodeUtils,
+  NoteUtils,
+  NoteProps,
+} from "@dendronhq/common-all";
+import type { Moment } from "moment";
+import moment from "moment";
+import { Calendar, Badge, Space } from "antd";
+import { blue } from "@ant-design/colors";
+import type { CalendarProps } from "antd";
+import _ from "lodash";
+import React, { useState, useMemo, useCallback, useEffect } from "react";
+import { DendronProps } from "../../lib/types";
+
+type AntdCalendarProps = CalendarProps<Moment>;
+
+function getMaybeDatePortion({ fname }: NoteProps, journalName: string) {
+  const journalIndex = fname.indexOf(journalName);
+  return fname.slice(journalIndex + journalName.length + 1);
+}
+
+const { EngineSliceUtils } = engineSlice;
+
+function CalendarView({ engine, ide }: DendronProps) {
+  // --- init
+  const ctx = "CalendarView";
+  const logger = createLogger("calendarView");
+
+  logger.info({
+    ctx,
+    state: "enter",
+    engine,
+    ide,
```

Comment:
for logging ide, since it contains a note object, you also don't want to log the note since it contains the note contents which is sensitive

Reply:
see [Logging](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.style.md#L17:L17)

Reply:
e6bf0a15